userbot/modules/notes.py

```diff
@@ -64,7 +64,7 @@
         )
 
 
-@register(incoming=True, pattern=r"#\w*", disable_edited=True)
+@register(incoming=True, outgoing=True, pattern=r"#\w*", disable_edited=True)
```

Comment:
```suggestion
@register(pattern=r"#\w*", disable_edited=True)
```
BTW, We don't need incoming=True and outgoing=True together, if we remove all of them it still will works well

Reply:
Ah i just recently found out this repo, so I didnt check the default settings without args being passed to the register decorator. I'll modify my fork to reflect these changes.